src/context.tsx

```diff
@@ -28,7 +28,7 @@ const initialState: State = {
     country: new Set()
   },
   data_page_state: {
-    mapOpen: true
+    mapOpen: false
```

Comment:
are we sure that we want the analyze tab to be open first? doesn't make that much sense to me